plugins/modules/cloud/lxd/lxd_profile.py

```diff
@@ -79,14 +87,14 @@
           - If not specified, it defaults to C($HOME/.config/lxc/client.key).
         required: false
         aliases: [ key_file ]
-        type: path
+        type: str
     client_cert:
         description:
           - The client certificate file path.
           - If not specified, it defaults to C($HOME/.config/lxc/client.crt).
         required: false
         aliases: [ cert_file ]
-        type: path
+        type: str
```

Comment:
Please revert this and the above change.

Reply:
I will restore the original text. Still, it's wrong. Python does not have a Path type and how the string is to be used can be found in the description.